SampleApps/CarthageTest/CarthageTest/ViewController.swift

```diff
@@ -18,7 +18,7 @@ class ViewController: UIViewController {
         let apiClient = BTAPIClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")!
 
         let amexClient = BTAmericanExpressClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
-        let applePayClient = BTApplePayClient(apiClient: apiClient)
+        let applePayClient = BTApplePayClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
In one of the PRs we can extract `sandbox_9dbg82cq_dcpspy2brwdjr3qn` into a property. Can be in a later one though.